datastream-utils/src/main/java/com/linkedin/datastream/common/zk/ZkClient.java

```diff
@@ -86,71 +80,20 @@ public ZkClient(String zkServers, int sessionTimeoutMs, int connectionTimeoutMs,
     _zkSessionTimeoutMs = sessionTimeoutMs;
   }
 
-  @Override
```

Comment:
What is the substitute for all the code that you are deleting?

Reply:
All those methods are implemented in https://github.com/apache/helix/blob/master/zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java .

We seemed to be doing basically the same thing as this implementation of ZkClient so I thought to simply rely on the base implementation for these.